.drone.yml

```diff
@@ -85,3 +85,24 @@ pipeline:
     when:
       branch: master
       event: push
+
+  unit_tests:
+    image: 795250896452.dkr.ecr.us-east-1.amazonaws.com/server-tig/${DRONE_REPO_NAME}:latest
+    commands:
+    - pip3 install poetry
+    - poetry install
+    - poetry run pytest --flake8 --isort --junitxml=test_output_junit.xml
+
+  check_formatting:
+    image: 795250896452.dkr.ecr.us-east-1.amazonaws.com/server-tig/${DRONE_REPO_NAME}:latest
+    commands:
+    - pip3 install poetry
+    - poetry install
+    - poetry run black --check --diff src tests
+
+  check_mypy:
+    image: 795250896452.dkr.ecr.us-east-1.amazonaws.com/server-tig/${DRONE_REPO_NAME}:latest
+    commands:
+    - pip3 install poetry
+    - poetry install
+    - poetry run pytest --mypy src
```

Comment:
We could probably move this stuff into config files so its just `poetry run pytest` for all the tests.

Reply:
Sure, this is just an exploration PR though